pkgs/top-level/all-packages.nix

```diff
@@ -9991,6 +9991,8 @@ with pkgs;
 
   lnch = callPackage ../tools/misc/lnch { };
 
+  lngen = callPackage ../applications/science/logic/lngen {};
```

Comment:
Please move the package to `pkgs/by-name/ln/lngen/package.nix` and remove this line, thanks.

Reply:
Thanks for the reviews. I moved the package :)